tools/cldr-apps/js/src/views/DashboardWidget.vue

```diff
@@ -55,6 +55,13 @@
           </template>
         </span>
         <span class="right-control">
+          <span
+            v-if="!includeOther"
+            class="cldr-nav-btn"
+            @click="reloadIncludeOther"
+          >
+            <input type="checkbox" />&nbsp;Other
```

Comment:
suggestion for future improvement: hover text for this control with brief explanation what it does, e.g., "Include notifications for ALL paths within the chosen coverage level that do not belong to another notification category. Note: the number of 'other' notifications may be very large"